apps/condo/domains/acquiring/schema/MultiPayment.js

```diff
@@ -153,7 +155,16 @@ const MultiPayment = new GQLListSchema('MultiPayment', {
         integration: ACQUIRING_INTEGRATION_FIELD,
     },
     hooks: {
-        // TODO (savelevMatthew): Create proper validations
+        validateInput: ({ resolvedData, context, addValidationError }) => {
+            if (!hasDvAndSenderFields(resolvedData, context, addValidationError)) return
+            const { dv } = resolvedData
+            if (dv === 1) {
+                // TODO (savelevMatthew): Write more complex validations
+                // NOTE: version 1 specific translations. Don't optimize this logic
```

Comment:
please use explicit returns

Reply:
Linter says it's redundant and we decided not to increase warning amount :)